physionet-django/console/templates/console/console_navbar.html

```diff
@@ -147,7 +147,7 @@
               <span class="nav-link-text">Legal</span>
             </a>
             <!-- submenu -->
-          {% if license_nav or dua_nav or code_of_conduct_nav %}
+          {% if license_nav or dua_nav or code_of_conduct_nav or perms.events.add_event %}
```

Comment:
This makes the menu uncollapsed by default. Need to change from the permission to the nav.

```suggestion
          {% if license_nav or dua_nav or code_of_conduct_nav or event_agreement_nav %}
```